### DISCUSSION TITLE
ld: library not found for -lemutls_w

### DISCUSSION THREAD

**Discussion by jianqixi** [idaholab/moose#21351] [Q&A Getting Started] [answered]
As a beginner of MOOSE, I have one error when installing the MOOSE in my Macbook. it showed that the library of -lemutls_w is missed, and then showed the following errors: clang-12: error: linker command failed with exit code 1 (use -v to see invocation). Thank you for your help,

**Comment by GiudGiud** [comment, marked as answer]
Hello

Please attach the full compilation log that shows the error.
Also if you could run the diagonistics script in moose/scripts and attach the output

btw, we use Github Discussions for such issues, please use this instead of issues. 

Thanks
Guillaume
